src/main/kotlin/racing/Car.kt

```diff
@@ -0,0 +1,34 @@
+package racing
+
+class Car private constructor(
+    private var sequence: Int = 0,
+    private var position: Int = 0,
+) : Vehicle {
+    fun getSequence(): Int {
+        return this.sequence
+    }
+
+    fun getPosition(): Int {
+        return this.position
+    }
```

Comment:
가변인 sequence와 position를 내부에서만 변경 가능하도록 하기 위해 private constructor로 숨겼군요! 👍

요걸 좀 더 개선한다면, 아래와 같이도 표현이 가능해보여요!

```suggestion
class Car : Vehicle {
    var sequence: Int = 0
        private set
    var position: Int = 0
        private set
```